Modules/core/Controller/CorespaceaccessController.php

```diff
@@ -337,6 +367,9 @@ public function pendingusereditAction($id_space, $id) {
             $modelSpace->setUserIfNotExist($pendingInfo["id_user"], $id_space, $form->getParameter("role"));
             $modelPending->validate($id, $_SESSION["id_user"]);
 
+            // multi-tenant feature: deleting user from core_pending_accounts table for this space in db
+            $modelPending->deleteByPendingAccountId($id_space, $id);
```

Comment:
invalid number of params
should not delete anyway, but use validated (or not) field afterward